extensions/wikia/AdEngine/ResourceLoaders/ResourceLoaderAdEngineSourcePointModule.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class ResourceLoaderAdEngineSourcePointModule extends ResourceLoaderAdEngineBase {
+	const TTL_SCRIPTS = 86400;   // one day for fresh scripts from SourcePoint
+	const TTL_GRACE = 3600; // one hour for old scripts (served if we fail to fetch fresh scripts)
+	const CACHE_BUSTER = 12;     // increase this any time the local files change
+	const REQUEST_TIMEOUT = 30;
```

Comment:
What do you think about decreasing this value? It could be also used in SevenOneMedia resources.